wordpress-mobile/WordPress-iOS
# Use PostRepository.getPost in Post History

What says in the title. The new `PostRepository` type was introduced in #21419.

> **Note**
> It might be easier to review by turning on the "Hide whitespace" option.

## Test Instructions

1. Write a new post and save it as draft.
2. Make a change to the post and save the change by clicking the "Update" button on the navigation bar.
3. Repeat the above step a few times.
4. Click the more button (`...`) and select "History"
5. Click one of the history and tap "Load". Verify the loaded history post content is correct.
6. Repeat and verify the above step a few times.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
The "Test Instructions"

3. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:


UI Changes testing checklist: N/A
